.maestro/setup_ui_tests.sh

```diff
@@ -8,14 +8,19 @@ source $(dirname $0)/common.sh
 
 # The simulator command requires the hyphens
 target_device="iPhone-16"
-target_os="iOS-18-1"
+target_os="iOS-18-2"
```

Comment:
FWIW, we can't select the minor version of iOS on Maestro Cloud, so we may end up being out of sync between local and CI. 
Anyway, this can happen in any case given the limited control we have, so I think it's good to keep selecting the minor version locally. It will help for consistency in out local development environment. We should just be aware we can't expect that consistency with cloud.